lib/hcloud/server.rb

```diff
@@ -10,7 +10,10 @@ class Server
       created: :time,
       server_type: ServerType,
       datacenter: Datacenter,
-      image: Image
+      image: Image,
+      iso: Iso,
+      private_net: [Network],
+      volumes: [Volume]
```

Comment:
This doesn't work, since the server's volume list contains only id's and not objects.

Reply:
Theoretically you can add some kind of future object in front which only gets filled with the id for example. So you can lazy load the real volume once an real volume key has been requested.